stake-pool/js/webpack.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Is this used?

Reply:
removed in https://github.com/solana-labs/solana-program-library/pull/1656/commits/0922acc76c05eaea559901e141034d2609db0a56